controllers/drplacementcontrol.go

```diff
@@ -1858,7 +1858,11 @@ func (d *DRPCInstance) ensureVRGManifestWorkOnClusterDeleted(clusterName string)
 	}
 
 	if d.ensureVRGIsSecondaryOnCluster(clusterName) {
-		err := d.mwu.DeleteManifestWorksForCluster(clusterName)
+		// delete VRG manifest work
+		mwName := d.mwu.BuildManifestWorkName(rmnutil.MWTypeVRG)
```

Comment:
The 3 lines can be folded into one: `err = d.mwu.DeleteManifestWork(d.mwu.BuildManifestWorkName(rmnutil.MWTypeVRG), clusterName)`

(in other places as well)